demos/object_detection_demo_ssd_async/include/yolov3.hpp

```diff
@@ -0,0 +1,58 @@
+#pragma once
```

Comment:
if we can handle not only tolo-v3 but also other variants it would be better to name file simple yolo.hpp